iosApp/iosApp/Localizable.xcstrings

```diff
@@ -1,49 +1,31 @@
 {
   "sourceLanguage" : "en",
   "strings" : {
-    "**%@** affected stops" : {
+    "**%ld** affected stops" : {
```

Comment:
question(non-blocking): can we preserve the translations we already have for the plural case here?

Reply:
I thought about it but since the original wasn't pluralized, I don't know which translations apply to the two/few/many/other forms for languages that have those, and filling in a single plural translation seems to fill in the remaining ones in english, which could potentially cause confusion/missed translations when we send them to the vendor.